addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -538,93 +482,40 @@
 
     // Kanban Grouped Layout - "Create new column" column
     .o_column_quick_create {
-        flex: 0 0 $o-kanban-small-record-width + 2*$o-kanban-group-padding;
-
-        .o_quick_create_folded {
-            cursor: pointer;
-            padding: 12px 16px;
-            white-space: nowrap;
-            font-weight: $font-weight-bold;
-            @include o-hover-opacity(.7);
-            @include o-hover-text-color($text-muted, $body-color);
-        }
-
-        .o_quick_create_unfolded {
-            margin: $border-width ($o-kanban-inside-hgutter * .5) 0;
-            padding: $o-kanban-inside-vgutter $o-kanban-inside-hgutter;
-
-            @include media-breakpoint-down(md) {
-                margin: 0;
-            }
-
-            .o_kanban_header {
-                height: 50px;
-
-                .o_kanban_examples {
-                    cursor: pointer;
-                }
-
-                input, input:focus, input:hover {
-                    font-size: 16px;
-                    background: transparent;
-                }
-            }
-
-            .o_kanban_muted_record {
-                background: map-get($grays, '300');
-                height: 70px;
-                margin: 10px 0px;
-            }
-
-        }
+        flex: 0 0 calc(var(--KanbanGroup__smallColumn-width, #{$o-kanban-small-record-width}) + (2 * var(--KanbanGroup__padding-h, #{$o-kanban-group-padding})));
     }
 
     @include media-breakpoint-down(md) {
         .o_kanban_group, .o_column_quick_create {
-            flex: 1 0 90%; // don't take full width to give a hint of next/previous column
+            flex-basis: 90%; // don't take full width to give a hint of next/previous column
             scroll-snap-align: center;
             overflow-y: scroll;
 
             .o_kanban_header {
                 position: sticky; // keep it visible
                 top: 0;
-                z-index: 10; // same as the sticky ControlPanel
+                z-index: $zindex-sticky;
```

Comment:
We definitively need responsive position utility-classes :/

Reply:
Bootstrap API ? 👀

Reply:
Yes, I'm on it